### DISCUSSION TITLE
Weird Debian 12 bookworm (gnome) behavior

### DISCUSSION THREAD

**Discussion by moustaleurie** [espanso/espanso#2080] [Q&A] [answered]
Hello everybody, I have installed Espanso using the instructions found in this [link](https://edafe.de/2023/09/how-to-install-espanso-from-source-on-debian-12-bookworm/).  It was the only way I managed to get it work. But it works! I only have two issues right now. The first is the acknowledged glitch with the sync tool whenever I change my keyboard layout - although, this is manageable! I learend to wait for 1 sec after the change and the sync tool window disappears. Here I want to talk about the second issue. I use Greek and English layout, if i trigger whatever using the English layout it works, but if I trigger it using the Greek layout, then it only works if in the replacement exists at least one Greek character! For example trigger: "~οτ" replace:"#![ot][/ot]" does not work, while replace:"#! α [ot][/ot]" works! It also works if I change one of the "ot" letters to greek letters. Does anybody know why is this happening and if there is a way to fix it?

**Comment by AucaCoyan** [comment, marked as answer]
Hi! I just saw your post, sorry for the delay! 😅.
Yeah, we have many problems with non english layouts. We have a tracking issue here:
https://github.com/espanso/espanso/issues/2123
We'll keep you posted there!
